views/windows.tmpl

```diff
@@ -17,62 +17,49 @@
-                    {{if ge $pendingWindows 100}}<option {{if eq $windowsCount 100}}selected{{end}} value="100">100 per page</option>{{end}}
-                    {{if eq $dropVal $windowsCount 20 30 50 100}}{{else}}<option value="{{$dropVal}}">{{$dropVal}} per page</option>{{end}}
-                    {{if eq $windowsCount 20 30 50 100 200}}{{else}}<option selected value="{{$windowsCount}}">{{$windowsCount}} per page</option>{{end}}
+                      <option {{if eq $windowsCount 10}}selected{{end}} value="10">10 per page</option>
+                      <option {{if eq $windowsCount 20}}selected{{end}} value="20">20 per page</option>
+                      <option {{if eq $windowsCount 30}}selected{{end}} value="30">30 per page</option>
+                      <option {{if eq $windowsCount 40}}selected{{end}} value="40">40 per page</option>
+                      <option {{if eq $windowsCount 50}}selected{{end}} value="50">50 per page</option>
+                      {{if eq $windowsCount 10 20 30 40 50}}{{else}}
+                          <option selected value="{{$windowsCount}}">{{$windowsCount}} per page</option>
+                      {{end}}
                     </select>
                 </span>
-                <nav aria-label="blocks navigation" data-limit="{{.Limit}}" class="ml-2 my-2 d-inline-block text-right">
-                  <ul class="pages mb-0">
-                    {{if ne .OffsetWindow 0}}
-                    <li>
-                        <a
-                        class="text-secondary"
-                        href="/ticketpricewindows?offset=0&rows={{.Limit}}"
-                        > Newest</a>
-                    </li>
-                    <li>
-                        <a
-                        class="text-secondary"
-                        href="/ticketpricewindows?offset={{ subtract .OffsetWindow .Limit }}&rows={{.Limit}}"
-                        > Newer</a>
-                    </li>
-                    {{end}}
-                    {{if lt $oldest $lastWindow}}
-                    <li>
-                        <a
-                        class="text-secondary"
-                        href="/ticketpricewindows?offset={{ add .OffsetWindow .Limit }}&rows={{.Limit}}"
-                        >Older</a>
-                    </li>
-                    <li>
-                        <a
-                        class="text-secondary"
-                        href="/ticketpricewindows?offset={{subtract $lastWindow .Limit}}&rows={{.Limit}}"
-                        >Oldest</a>
-                    </li>
-                    {{end}}
-                  </ul>
-                </nav>
+                <span class="fs12 nowrap text-right">
+                    <select
+                        data-target="pagenavigation.pageage"
+                        data-action="change->pagenavigation#setPageAge"
+                        data-rows="{{.Limit}}"
+                        data-offsetkey="offset"
```

Comment:
Comment about offset also applies here.

Reply:
this will no longer be needed as it's going back to being buttons